high_energy_opt/highe_opt.yaml

```diff
@@ -1,73 +0,0 @@
-xopt:
```

Comment:
Why was this deleted? It's an old optimization, but can go in a different folder?

Reply:
Same question for some of the older yaml files / dmd images?